R/render.R

```diff
@@ -306,9 +327,9 @@ use_lang_chapter <- function(chapters_list, language_code, book_name, directory)
   chapters_list
 }
 
-add_links <- function(path, main_language = main_language,
-                     language_code, site_url, type, config, output_folder,
-                     path_language) {
+add_links <- function(path, main_language, # nolint: cyclocomp_linter
```

Comment:
As suggested, I'll have a look at this when I get a chance. I think I can break it up into smaller parts.